en/_drafts/subsurface-4-9-4-released.MD

```diff
@@ -38,7 +38,7 @@ This release fixes a number of issues in {{ page.previousVersion }} and adds som
 
   - planner: improve rounding of stop durations in planner notes
 
-# bugfies:
+# bugfixes:
```

Comment:
funny, I had noticed this while playing around with things and then forgot to go back to fix. Good catch. Thanks